airbyte-integrations/connectors/destination-redshift/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 }
```

Comment:
I think we are missing a redshift metadata version bump

Reply:
I think that the build is not fixed yet so I didn't add it on purpose.

Reply:
Redshift was not publish with the 30 MB bump because of the build issue